Assets/MixedRealityToolkit/Interfaces/Devices/IMixedRealityHandVisualizer.cs

```diff
@@ -5,7 +5,7 @@
 using Microsoft.MixedReality.Toolkit.Input;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.Devices
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input
```

Comment:
None of these are specific to WindowsMixedReality, so they should probably be moved into just `Toolkit.Input`, with the rest of the interfaces and profiles.